db/schema.rb

```diff
@@ -19,7 +19,7 @@
   create_table "activities", force: :cascade do |t|
     t.integer  "appointment_id",              null: false
     t.integer  "user_id"
-    t.string   "message",        default: ""
+    t.string   "message",        default: "", null: false
```

Comment:
How did this creep in? If this is down to someone borking the migrations / schema I think we should promote this to a separate PR.